spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -219,6 +219,10 @@ def tmp2ofolder(self):
                  os.path.join(self.path_ofolder, self.tissue_bridges_png_name))
 
     def pack_measures(self):
+
+        # Replace 'nan' with 'n/a' ('nan' results in empty cells in Excel)
+        self.measure_pd = self.measure_pd.replace(np.nan, 'n/a')
```

Comment:
> was there a specific issue with having nan/empty cells, rather than 'n/a'?

Good question! I think I just wanted to avoid empty cells in Excel:

<img width="700" alt="image" src="https://github.com/spinalcordtoolbox/spinalcordtoolbox/assets/39456460/834cfe13-6ce2-43ab-ac0f-cde64e35dddb">

I tried to comment the line in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4489/commits/1d4641408b269d71e27c59177fe64b2aac38ec29 to see whether the tests pass

Reply:
It looks like the tests pass! 

To me, empty cells seem clear enough to me? But, maybe there is something I'm missing.

Reply:
I think empty cells are not ideal for CSV, for example, when printing CSV in CLI using `cat.` But here, as we use XLS, we can go with empty cells, I hope.